### ISSUE TITLE
Debug step by step/ breakpoint bit in vscode

### ISSUE THREAD

**Issue by EnricoBeltramo** [teambit/bit#6264]
### Description
Is it possible to debug bit component with vscode? I mean set a breakpoint, start debug, and move step by step?
If I run the debug in bit, it do some test and after detach debugger, without enter in a breakpoint

**Comment by GiladShoham** [comment]
please try to follow the instructions here https://bit.dev/docs/getting-started/installing-bit/editor-setup#debugging-in-vscode

**Comment by EnricoBeltramo** [comment]
Yes, effectively how I said, I used that launch file proposed in https://bit.dev/docs/getting-started/installing-bit/editor-setup#debugging-in-vscode, but it doesn't allow to debug step by step.
I was able to debug with this configuration:

{
  // Use IntelliSense to learn about possible attributes.
  // Hover to view descriptions of existing attributes.
  // For more information, visit: https://go.microsoft.com/fwlink/?linkid=830387
  "version": "0.2.0",
  "configurations": [
    {
      "type": "pwa-chrome",
      "request": "launch",
      "name": "Launch Chrome against localhost",
      "url": "http://localhost:3000",
      "webRoot": "${workspaceFolder}"
    }
  ]
}

So my question is: what is mean to do the proposed launch.json if is not possible to debug step by step? May be it needs some additional configuration of vscode side?
Thank
